README.md

```diff
@@ -28,16 +28,31 @@ Demonstrate your understanding of this week's concepts by answering the followin
 
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
-1. Describe the biggest difference between `.forEach` & `.map`.
+1. Describe the biggest difference between `.forEach` & `.map`. 
+
+.map creates a new array, .forEach returns undefined
 
 2. What is the difference between a function and a method?
 
+A method is a function inside a class constructor
+
 3. What is closure?
 
+Manipulating variables in such a way that each new iteration of a function doesn't destroy the cumulative variable value (like in a counter)
+
```

Comment:
storing variables in the outer context, made available to the funcitons utilizing them.